modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -1,10 +1,12 @@
 // Module included in the following assemblies:
 
-// OpenStack
-// * networking/load-balancing-openstack.adoc
 // Bare metal
 // * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 // * installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc
+// OpenStack
+// * networking/load-balancing-openstack.adoc
+// Nutanix
```

Comment:
Do I need to specify that a user-managed LB is possible as a Day-2 task for Nutanix and OpenStack?

Reply:
hmm, I don't think we need to add the day2 task if no before.